lib/react_on_rails/helper.rb

```diff
@@ -91,6 +91,36 @@ def react_component(component_name, options = {})
       end
     end
 
+    def stream_react_component(component_name, options = {})
+      options = options.merge(stream?: true)
+      result = internal_react_component(component_name, options)
+      build_react_component_result_for_server_streamed_content(
+        rendered_html_stream: result[:result],
+        component_specification_tag: result[:tag],
+        render_options: result[:render_options]
+      )
+    end
+
+    def stream_react_component_async(component_name, options = {})
```

Comment:
To see how this function is used look [here](https://github.com/shakacode/react_on_rails_pro/pull/407/files#diff-7bbea53e72fefe60558edaf0ec093a01425ed7403fc60901320a7c3d9e154f8dR1) and [here](https://github.com/shakacode/react_on_rails_pro/pull/407/files#diff-4255e4faea72596818e5393cfddc705c9f7a72b4b1a77fef16b8835cfa387216R31)